backend/services/office_hours/event.py

```diff
@@ -0,0 +1,679 @@
+                the office hours section associated with the event (`oh_event`).
+        """
+        # PERMISSIONS: Check If User is an OH Section Member - Exception If Not.
+        self._check_user_section_membership(subject.id, oh_event.oh_section.id)
+
+        # Queued Tickets
+        queued_ticket_entities = self._get_queued_tickets_by_oh_event(oh_event.id)
+        queued_ticket_count = len(queued_ticket_entities)
+
+        # Called Tickets
+        called_ticket_entities = self._get_called_tickets_by_oh_event(oh_event.id)
+        called_ticket_count = len(called_ticket_entities)
+
+        # Build Event Status
+        event_status = OfficeHoursEventStatus(
+            open_tickets_count=called_ticket_count,
+            queued_tickets_count=queued_ticket_count,
+        )
+
+        # Return the Status model
+        return event_status
+
+    def get_event_queue_stats_for_student_with_ticket(
+        self, subject: User, oh_event: OfficeHoursEvent, ticket_id: int
+    ) -> StudentOfficeHoursEventStatus:
+        """Retrieve queued and called ticket statistics for a specific office hours event.
+
+        Args:
+            subject (User): The user object representing the authenticated user making the request.
+            oh_event (OfficeHoursEvent): The details of the office hours event.
+
+        Returns:
+            OfficeHoursEventStatus: An `OfficeHoursEventStatus` object representing the statistics
+                (queued and called ticket counts) for the specified office hours event.
+
+        Raises:
+            PermissionError: Raised if the authenticated user (`subject`) is not a member of
+                the office hours section associated with the event (`oh_event`).
+        """
+        # PERMISSIONS:
+
+        # 1. Check If User is an OH Section Member - Exception If Not.
+        current_user_section_member_entity = self._check_user_section_membership(
+            subject.id, oh_event.oh_section.id
+        )
+
+        # 2. Check Ticket Exists in Office Hours Event
+        ticket_entity = self._session.get(OfficeHoursTicketEntity, ticket_id)
+
+        if ticket_entity is None:
+            raise ResourceNotFoundException(f"Ticket id={ticket_id} doesn't exist.")
+
+        if ticket_entity.oh_event_id != oh_event.id:
+            raise Exception(
+                f"Given Ticket id={ticket_id} Is Not A Part of OH Event id={oh_event.id}"
+            )
+
+        # 3. Check if User is Creator of Ticket
+        ticket_creators = ticket_entity.to_details_model().creators
+        is_creator = False
+
+        for creator in ticket_creators:
+            if creator.id == current_user_section_member_entity.id:
+                is_creator = True
+
+        if not is_creator:
+            raise PermissionError(
+                f"User Doesn't Have Permission to View Queue Stats for Ticket id={ticket_entity.id}"
+            )
```

Comment:
Small nit: For conciseness, I believe we can condense this:
```py
is_creator = current_user_section_member_entity.id in [creator.id for creator in ticket_creators]
if not is_creator:
    raise PermissionError("...")
```

Alternatively, we can also probably do something like:
```py
if current_user_section_member_entity.id not in [creator.id for creator in ticket_creators]:
    raise PermissionError("...")
```

Not sure which is more readable!